scss/components/localization-editor.scss

```diff
@@ -29,11 +29,6 @@ $block: #{$fd-namespace}-localization-editor;
         }
     }
 
-    &__button {
-        height: 100% !important;
-        max-height: 100% !important;
-    }
-
```

Comment:
So, I agree this seems odd to have here and also with the `!important` declarations, however, simply removing this causes issues when hovering or focusing on the button.  This _seems_ like something that should be fixed within input groups rather than here.  I am seeing the same weirdness on hover and focus on buttons within an input group.

Reply:
Yeah my plan was to leave this hanging until we have time to resolve the input group hover/focus problems. I remember having this issue in nui-widgets, especially with the border hovers on timepicker and whatnot, but the code is so different the solution we came up with there didn't seem to apply.